LICENSE-MIT

```diff
@@ -0,0 +1,23 @@
+
+Copyright (c) 2015 Davide Di Criscito.
```

Comment:
Well, remove my name from the license and add yours! Licenses must always be provided!